services/actions/context.go

```diff
@@ -160,3 +165,19 @@ func mergeTwoOutputs(o1, o2 map[string]string) map[string]string {
 	}
 	return ret
 }
+
+func (g *GiteaContext) ToGitHubContext() *model.GithubContext {
+	ghCtx := &model.GithubContext{}
+
+	gitCtxRaw, err := json.Marshal(g)
+	if err != nil {
+		log.Error("ToGitHubContext.json.Marshal: %v", err)
+	}
+
+	err = json.Unmarshal(gitCtxRaw, ghCtx)
```

Comment:
It is quite fragile by using Marshal+Unmarshal, it's unclear what value a field would get.

To make it stable, I think we should assign the fields one by one explicitly.